fs/ensure_symlink.ts

```diff
@@ -5,7 +5,7 @@ import { ensureDir, ensureDirSync } from "./ensure_dir.ts";
 import { getFileInfoType, type PathType } from "./_get_file_info_type.ts";
 import { toPathString } from "./_to_path_string.ts";
 
-const isWindows = Deno.build.os === "windows";
+const isWindows = typeof window === "undefined" && Deno.build.os === "windows";
```

Comment:
`window` object is defined in Deno 1.x, and I think it's probably too early to stop supporting Deno v1.x.

Let's do something like `(globalThis as any).Deno?.build.os === "windows"` instead, similarly to `@std/path` package. ref. https://github.com/denoland/std/blob/ae7048f9ca983f038174aca4cd89c120945aef23/path/_os.ts#L7C3-L7C51

Reply:
Done

Reply:
Is `as any` required? It's causing tests to fail.